### DISCUSSION TITLE
Statistics! (3.2020-3.2021)

### DISCUSSION THREAD

**Discussion by westnordost** [streetcomplete/StreetComplete#2671] [General]
# Number of Users
installed via **Google Play**: 9709
installed via **F-Droid** or elsewhere: ca. 3500

The following statistics only include users from Google Play, as there are no statistics of F-Droid users. The number of F-Droid users is only an estimation.

## Over the last year

![installed_audience_2020](https://user-images.githubusercontent.com/4661658/112069178-aa4f0900-8b6b-11eb-8634-eb0e6f0b8a6b.png)

Also shown is the growth of users from the top 3 countries with most users.

# Active Users

# Monthly Active Users

![monthly_active_users_2020](https://user-images.githubusercontent.com/4661658/112069774-b6879600-8b6c-11eb-9c29-ef71fcf4df34.png)

Also shown is the growth of monthly active users of the top 1 country (Germany) because I couldn't select any other country in the dashboard. 🤷 
The statistic is only recorded since October.

# Map Views

![map_views](https://user-images.githubusercontent.com/4661658/112074033-9c9e8100-8b75-11eb-8c02-15554864502d.png)

This statistics is actually the statistic how many tiles are requested from the map tile provider Jawg.io. The spikes are on the weekend, most people use it then. Not too surprising.
Bit surprising though is that the use of the app is apparently not strongly connected to the weather / season. I'd have thought so.

# Edits done

![image](https://user-images.githubusercontent.com/4661658/112072594-77f4da00-8b72-11eb-9eb7-36e1b3d844eb.png)

Edits done in 2019 compared with edits done in 2020.

# User Location

## Per Region

![location_of_users_continent_2020](https://user-images.githubusercontent.com/4661658/112069952-1aaa5a00-8b6d-11eb-974e-c027d5098b1b.png)

## Per Country

![location_of_users_2020](https://user-images.githubusercontent.com/4661658/112069907-fcdcf500-8b6c-11eb-873e-607e6eccdd6f.png)

Shown are only countries with more than 100 users.

## As Table

| Country | User Count |
|---------------------------|------|
| **Germany**                   | 2281 |
| **France**                    | 1084 |
| **US**                        | 1064 |
| **UK**                        | 613  |
| **Poland**                    | 383  |
| **Italy**                     | 310  |
| **Spain**                     | 300  |
| **Netherlands**               | 238  |
| **Denmark**                   | 232  |
| **Switzerland**               | 228  |
| **Australia**                 | 215  |
| **Belgium**                   | 200  |
| **Canada**                    | 183  |
| **Russia**                    | 176  |
| **Austria**                   | 172  |
| **Sweden**                    | 161  |
| **Finland**                   | 136  |
| **Czechia**                   | 133  |
| **Portugal**                  | 77   |
| **Iran**                      | 77   |
| **Democratic Republic of the Congo** | 75   |
| **Norway**                    | 74   |
| **Ukraine**                   | 66   |
| **Ireland**                   | 64   |
| **India**                     | 59   |
| **Brazil**                    | 58   |
| **Hungary**                   | 56   |
| **New Zealand**               | 53   |
| **Japan**                     | 46   |
| **Argentina**                 | 44   |
| Romania                   | 43   |
| **Mexico**                    | 34   |
| **Slovakia**                  | 28   |
| Latvia                    | 28   |
| **Lithuania**                 | 27   |
| **Greece**                    | 27   |
| **Chile**                     | 27   |
| **Croatia**                   | 24   |
| Israel                    | 23   |
| **Turkey**                    | 22   |
| **Colombia**                  | 22   |
| **Taiwan**                    | 20   |
| **Bolivia**                   | 20   |
| **Tunisia**                   | 19   |
| **Singapore**                 | 18   |
| Belarus                   | 18   |
| **South Korea**               | 16   |
| Philippines               | 16   |
| Thailand                  | 15   |
| South Africa              | 15   |
| Slovenia                  | 15   |
| **Bulgaria**                  | 15   |
| Myanmar                   | 14   |
| **Luxembourg**                | 13   |
| **Ghana**                     | 12   |
| Estonia                   | 12   |
| Bangladesh                | 12   |
| **Serbia**                    | 11   |
| **Malaysia**                  | 11   |
| Saudi Arabia              | 10   |
| Vietnam                   | 9    |
| UAE                       | 9    |
| **Réunion**                   | 9    |
| **Indonesia**                 | 9    |
| **Hong Kong**                 | 9    |
| **Uruguay**                   | 8    |
| **Peru**                      | 8    |
| Egypt                     | 8    |
| **Guatemala**                 | 7    |
| Algeria                   | 7    |
| Morocco                   | 6    |
| Malta                     | 6    |
| **China**                     | 6    |
| **Venezuela**                 | 5    |
| Togo                      | 5    |
| Other (less than 5 users) | 139  |

Countries In bold: The app is translated in at least one official language of that country. 

Of the "trade languages" (i.e. spoken by many people even if it is not their first language), the one that is most missing is Arabic. Might be one reason for the low usage numbers in Northern Africa and West Asia.

**Comment by westnordost** [comment]
Statistics for 2019 are at #1416

**Comment by smichel17** [comment]
User count would be interesting to know on the leaderboards. # 60/1064 is a more interesting stat than just # 60.

edit: also wow that's a lot more edits!

**Comment by RubenKelevra** [comment]
@westnordost wrote
> Bit surprising though is that the use of the app is apparently not strongly connected to the weather / season. I'd have thought so.

I think it's just connected to precipitation, and since the app is used in so many different regions there's hardly any trend to be seen :)

**Comment by westnordost** [comment]
I noticed that I can simply read the edit count per months from the statistics table using
```SQL
SELECT DATE_FORMAT(created_at,'%Y-%m'), SUM(solved_quest_count)
FROM changesets
GROUP BY YEAR(created_at), MONTH(created_at)
```

Result:
![edit_history](https://user-images.githubusercontent.com/4661658/119569596-3c170480-bdaf-11eb-8b4e-010433b41de1.png)

**Reply by kmpoppe** [reply]
Almost 1 Million Changesets! That's a pretty big milestone coming up!

**Reply by HolgerJeromin** [reply]
Do you know where the bump in November 2020 came from?

**Reply by matkoniecz** [reply]
Maybe

> Wait for the download no more! 
> Reworked it completely. Quest download is now substantially faster and the pins for all quest types appear immediately at the same time!

upgrade? 

https://github.com/streetcomplete/StreetComplete/releases/tag/v26.0

It made SC much nicer.

**Reply by HolgerJeromin** [reply]
All user have more quests to solve. Yes, could be a good explanation.

**Reply by peternewman** [reply]
I wonder if that implies that a lot of people are using it offline or only uploading quests on demand?

I was certainly surprised to see a road fairly nearby to me didn't have pavement quest answered until I realised that before some of these features, and particularly v32.0, if offline it would need about three trips to get there, between surface, lit and pavement.

**Reply by peternewman** [reply]
Also I'd be surprised if the pandemic didn't cause some of the lift in 2020/2021, but I guess we'll see over the course of the next year. Being legally mandated to stay at home and with little else to do, I'd imagine more people have been going for more walks and cycles than they used to, and possibly using SC while doing so.

**Reply by smichel17** [reply]
I have answered a lot of building type quests in my area. Before v26, I had to manually scan for quests ~4 times in the same area before all the building type quests would appear (because only a few quest types were downloaded per scan, and building-type was relatively low priority). This might take a few minutes, depending on how big of an area I needed to download. I suspect many people either did not realize this or did not have the patience to do it.

**Comment by westnordost** [comment]
I made a better statistic that also looks at the active users per month and annotated the verions released.

<details><summary>SQL statement</summary>

```sql
SELECT 
  DATE_FORMAT(created_at,'%Y-%m') as 'Date',
  SUM(solved_quest_count) as 'Edits',
  COUNT(DISTINCT user_id) as 'Users'
FROM changesets
GROUP BY 
  YEAR(created_at),
  MONTH(created_at) 
```
</details>

![image](https://user-images.githubusercontent.com/4661658/119691059-8b5e4300-be4a-11eb-8d85-94e987e90d1c.png)

**Reply by smichel17** [reply]
So, it looks like ~85% of installs are from the Play Store? I see ~4000 installs reported for January 2021 on the Play Store graph and ~4800 in this one. Although I guess this only counts users who uploaded at least one edit, while Play Store counts all installs?

**Reply by westnordost** [reply]
Correct
